suites/squid/nvmeof/tier-3_8-1-nvmeof_4-group_8-gw_scale.yaml

```diff
@@ -0,0 +1,519 @@
+                    - node20
+              pos_args:
+                - nvmeof_pool
+                - group2
+          - config:                        # Group 3 with no In band authentication - to be used for vmware testing
+              command: apply
+              service: nvmeof
+              args:
+                placement:
+                  nodes:
+                    - node21
+                    - node22
+                    - node23
+                    - node24
+                    - node25
+                    - node26
+                    - node27
+                    - node28
+              pos_args:
+                - nvmeof_pool
+                - group3
+          - config:                       # Group 4 with no spdk huge pages
+              command: apply_spec
+              service: orch
+              validate-spec-services: true
+              specs:
+                - service_type: nvmeof
+                  service_id: nvmeof_pool.group4
+                  placement:
+                    nodes:
+                      - node29
+                      - node30
+                      - node31
+                      - node32
+                      - node33
+                      - node34
+                      - node35
+                      - node36
+                  spec:
+                    pool: nvmeof_pool
+                    group: group4
+                    spdk_mem_size: 4096
+      desc: deploy NVMeoF service for 4 GW groups with 8 GWs
+      destroy-cluster: false
+      do-not-skip-tc: true
+      module: test_cephadm.py
+      name: deploy 4 NVMeoF services
+      polarion-id: CEPH-83595696
+
+# Group 1 configuration  - All 8.1 features
+  - test:
+      abort-on-fail: true
+      config:
+        node: node4
+        rbd_pool: rbd1
+        do_not_create_image: true
+        rep-pool-only: true
+        steps:
+          - config:
+              service: subsystem
+              command: add
+              args:
+                subsystems: 76
+                max-namespaces: 2048
+          - config:
+              service: listener
+              command: add
+              args:
+                subsystems: 76
+                port: 4420
+                group: group1
+                nodes:
+                  - node4
+                  - node5
+                  - node6
+                  - node7
+                  - node8
+                  - node9
+                  - node10
+                  - node11
+                  - node12
+          - config:
+              service: host
+              command: add
+              args:
+                subsystems: 76
+                group: group1
+      desc: GW group with 8 GWs and 76 subsystems
+      destroy-cluster: false
+      module: test_ceph_nvmeof_gateway_sub_scale.py
+      name: Configure 76 subsystems on group1
+      polarion-id: CEPH-83595512                     # This shall be replaced with In band authentication functionality at scale
+
```

Comment:
In this case, are we assume any test case defined below should be using above service and modify their test case accordingly?

Reply:
I think it would be confusing to others to add their test cases considering what subsystems and other entities they need to work with.

Reply:
Yes, any test case confined to that GW group must be modified accordingly - especially scale numbers capped to 60% of max limit. This is what decided in the meeting